Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:15-alpine3.13
+
+# iproute2 includes the ss cli
+RUN apk update && apk add iproute2
+
+WORKDIR /opt/sidecar-monitor
+COPY package.json .
+COPY src .
+
+RUN npm install
```

Comment:
Move this to line 8 to optimize docker builds. That will cause the result of `npm install` to stay cached by docker until a new dependency is added.